test/src/test/java/org/corfudb/integration/AbstractIT.java

```diff
@@ -436,4 +439,20 @@ public Process runServer() throws IOException {
             return corfuServerProcess;
         }
     }
+
+    /**
+     * A helper method that takes host and port specification, start a single server and returns a process.
+     *
+     * @param host Hostname/IP of the server.
+     * @param port Port of the server.
+     * @return Process instance of the server.
+     */
+    Process runSinglePersistentServer(String host, int port) throws IOException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651455)